.devcontainer/scripts/postCreateCommand.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# Copyright (c) 2023 Robert Bosch GmbH
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Clone python template into devContainer to act as a testing playground for CLI development
+echo "#######################################################"
+echo "###        Clone vehicle-app-python-template        ###"
+echo "###         to act as a testing playground          ###"
+echo "###               for CLI execution                 ###"
+echo "#######################################################"
+
+rm -rf vehicle-app-repo/
+git clone https://github.com/eclipse-velocitas/vehicle-app-python-template.git vehicle-app-repo
```

Comment:
```suggestion
git clone --depth=1 https://github.com/eclipse-velocitas/vehicle-app-python-template.git vehicle-app-repo
```

This will perform a shallow clone and speed up the cloning as we do not need the entire history of the repository.